qemu/tests/win_irq_check.py

```diff
@@ -34,17 +48,31 @@ def irq_check(session, device_name, devcon_folder, timeout):
         hwids = win_dev.get_hwids(session, device_name, devcon_folder, timeout)
         if not hwids:
             test.error("Didn't find %s device info from guest" % device_name)
+        if params.get("check_vectors") == "yes":
+            vectors = int(get_vectors_fqtree())
```

Comment:
Better:
`if params.get("check_vectors", "no") == "yes"  `
that declare the default value of check_vetors is "no", not None.

Reply:
updated